### DISCUSSION TITLE
HACK

### DISCUSSION THREAD

**Discussion by HACKERParPar** [community/community#17709] [GitHub Education] [answered]
GitHub Education wants to make it easier for students to get their start in the open source community. We are looking for student leaders who are interested in creating repositories to teach other students new skills or develop an open source project with other students. These repositories will live in GitHub, but they will also be exclusively showcased to the student community on GitHub Global Campus.

**Comment by singhlify** [comment]
Hi! I'm interested, can you please provide some more details.

**Reply by Osama669** [reply]
وأنا مهتم أيضاً

**Comment by Akpeters** [comment]
Am also interested.
just want to be an hacker

**Comment by smashcoder2003** [comment]
I am interested and looking forward to working with **other members** and _help them_

**Comment by d-coder111** [comment, marked as answer]
> GitHub Education wants to make it easier for students to get their start in the open source community. We are looking for student leaders who are interested in creating repositories to teach other students new skills or develop an open source project with other students. These repositories will live in GitHub, but they will also be exclusively showcased to the student community on GitHub Global Campus.

Check out this https://education.github.com/students

**Comment by Vikash-8090-Yadav** [comment]
hii greetings of the day !!
can u please tell when the trading card should be dilievered in india 
i submitted on 2 may 22
it sucessfully merged also .
thank u

**Comment by Sulagna-Dutta-Roy** [comment]
@HACKERParPar I am Interested in this global campus, how to Join as a leader??
